df.world.xml

```diff
@@ -993,9 +993,7 @@
         <compound name='enemy_status_cache'> bay12: unit_reaction_handlerst
             <static-array name='slot_used' type-name='bool' count='500'/>
             <static-array name='rel_map' count='500'>
-                <static-array count='500'> bay12: unit_reactionst
-                    <enum name='reaction' type-name='unit_reaction_type'/>
-                </static-array>
+                <static-array type-name='unit_reaction_type' count='500'/> bay12: unit_reactionst
```

Comment:
This one was actually intentional - it's not a 500x500 array of enums, but a 500x500 array of **structures** which happen to contain nothing but an enum.

Reply:
the compiler didn't seem to like it. it couldn't initialize/address it properly